Catalog/README.md

```diff
@@ -0,0 +1,52 @@
+# Low Code App Editor
+
+[![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?project=SkylineCommunications_Low-Code-App-Extensions&metric=alert_status)](https://sonarcloud.io/summary/new_code?id=SkylineCommunications_Low-Code-App-Extensions)
+
+The **Low Code App Editor** simplifies managing low-code applications within the DataMiner System. It allows you to import, export, edit, and delete apps while offering customization options for metadata, resources, and deployment settings. With features like tailored export configurations and cross-Agent synchronization, the tool streamlines app life cycle tasks, making it a practical solution for maintaining and deploying low-code apps efficiently.
+
+> [!IMPORTANT]
+> After the deploy you'll need to manually replace on of the references.
+>
+> C:\Skyline DataMiner\ProtocolScripts\WebApiLib.dll needs to be C:\Skyline DataMiner\Webpages\API\bin\WebApiLib.dll
```

Comment:
Do I understand it correctly that this "important" block mentions the same information as step 2 under "Installation" below? In that case, it might be better to leave out this block here, because it's duplicate info and it might have a slightly discouraging effect on users when this is the first thing they see when reading about the app.

Reply:
Jarno gave me a good idea to completely get rid of this step. But I'll need to check if it gives any implications on the DataMiner system elsewhere.

I added the important block, because I already got a few people texting me saying the script doesn't start and when I asked if they checked the install procedure they said no and then when they change de reference everything works as expected so that's why I wanted it at the top. 

But hopefully Jarno's idea works as expected and the step and warning can be removed completely.

Reply:
Jup it's working as expected, since Step 2 is then no longer needed and the warning can also be removed. Should the installation part then also be completely removed, because the steps are now just 'click the deploy button'?

Reply:
I like that so instead of an Installation part it would be a "Getting started" section instead